pkg/sriov/operatorConfig.go

```diff
@@ -0,0 +1,89 @@
+package sriov
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+
+	srIovV1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OperatorConfigBuilder provides struct for SriovOperatorConfigBuilder object which contains connection to cluster and
+// SriovOperatorConfig definitions.
+type OperatorConfigBuilder struct {
+	// Dynamically discovered SriovOperatorConfig object.
+	Objects *srIovV1.SriovOperatorConfig
+	// apiClient opens api connection to the cluster.
+	apiClient *clients.Settings
+	// nodeName defines on what node SriovOperatorConfig resource should be queried.
+	nodeName string
+	// nsName defines SriovOperatorConfig operator namespace.
+	nsName string
+	// errorMsg used in discovery function before sending api request to cluster.
+	errorMsg string
+}
+
+// NewOperatorConfigBuilder creates new instance of SriovOperatorConfig.
+func NewOperatorConfigBuilder(apiClient *clients.Settings, nameConfig, nsname string) *OperatorConfigBuilder {
```

Comment:
It looks line you are going to change this object.
Please remove discover func and NewOperatorConfigBuilder and replace it by PullOperatorConfigBuilder

Reply:
structure removed as unnecessary